src/helpers/adm-zip-helpers.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Returns a list of top-level directories in the zip file
+ * @param {IZipEntry[]} zipEntries 
+ * @returns {string[]} list of top-level directories
+ */
+function getTopLevelDirectories(zipEntries) {
```

Comment:
Not a fan of "helpers". Sounds like "heap of random things" to me. No need to change that just wanted to remark my sentiment.

Reply:
And that's what they are... ;) I'l move this particular helper to a better location when I migrate the validator to yauzl.